templates/js/logo.js

```diff
@@ -0,0 +1,29 @@
+(function setLogoColor() {
+
+  function getColor(view) {
+    switch(view) {
+    case 'movies':
+      return 'svg-fill--light-blue';
```

Comment:
I think these are already defined? Are they the secondary colours?

emily has some very similar code somewhere, check there, make the style consistent and dry out if possible